### ISSUE TITLE
Session cookie and yii\web\Request::getCookies()

### ISSUE THREAD

**Issue by SDKiller** [yiisoft/yii2#6867]
Related https://github.com/yiisoft/yii2/issues/5480#issuecomment-58664284

IMO, makes sense to skip session cookie from validation in `loadCookies()` and include it in `CookieCollection` - to make retrieving cookies more straightforward.

It is inconvenient to keep in mind that you need different approach to get complete cookies list. 
It will only result that everybody will access `$_COOKIE` directly.

**Comment by dynasource** [comment]
makes sense to collect cookies inside CookieCollection and to merge the $cookie parameters of Request into 1 array. Cleans up the Request object.

**Comment by samdark** [comment]
2.1?
